ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -159,6 +163,250 @@ objects:
+              get machines \
+              -l machine.openshift.io/cluster-api-machine-role=master \
+              -o jsonpath='{range .items[*]}{.status.addresses[?(@.type=="InternalIP")].address}{"\n"}{end}' || true))
+            if [[ "${#NEW_MASTER_IPS[@]}" == "3" ]]; then break; fi
+            sleep 30
+          done
+          oc get machines -n openshift-machine-api
+          if [[ "${#NEW_MASTER_IPS[@]}" != "3" ]]; then
+            echo "${NEW_MASTER_IPS[@]}"
+            exit 1
+          fi
+
+          echo "Verify new master nodes have joined the cluster"
+          FOUND_MASTERS=0
+          for i in $(seq 1 60)
+          do
+            FOUND_MASTERS=($(oc --request-timeout=5s get nodes -l node-role.kubernetes.io/master= -o name --no-headers || true))
+            if [[ "${#FOUND_MASTERS[@]}" == "3" ]]; then break; fi
+            sleep 30
+          done
+          oc get nodes
+          if [[ "${#FOUND_MASTERS[@]}" != "3" ]]; then
+            echo "${FOUND_MASTERS[@]}"
+            exit 1
+          fi
+
+          echo "Update DNS and LB"
+          # aws cli magic
+          easy_install --user 'setuptools==12.0'
+          easy_install --user boto3
+          cat > /tmp/update_route_53.py <<- PYTHON_EOF
+        import boto3
+        import os
+        import sys
+
+        if len(sys.argv) < 3:
+            print("Usage: ./update_route_53.py <RECORD> <IP>")
+            sys.exit(1)
+
+        record = sys.argv[1]
+        ip = sys.argv[2]
+        print("record: %s" % record)
+        print("ip: %s" % ip)
+
+        domain = "%s.%s" % (os.environ["CLUSTER_NAME"], os.environ["BASE_DOMAIN"])
+
+        client = boto3.client('route53')
+        r = client.list_hosted_zones_by_name(DNSName=domain, MaxItems="1")
+        zone_id = r['HostedZones'][0]['Id'].split('/')[-1]
+
+        response = client.change_resource_record_sets(
+            HostedZoneId=zone_id,
+            ChangeBatch= {
+                'Comment': 'add %s -> %s' % (record, ip),
+                'Changes': [
+                {
+                    'Action': 'UPSERT',
+                    'ResourceRecordSet': {
+                        'Name': record,
+                        'Type': 'A',
+                        'TTL': 60,
+                        'ResourceRecords': [{'Value': ip}]
+                    }
+                }]
+        })
+        PYTHON_EOF
+          for i in "${!NEW_MASTER_IPS[@]}"; do
+            ETCD_NAME="etcd-${i}.${CLUSTER_NAME}.${BASE_DOMAIN}"
+            python /tmp/update_route_53.py "${ETCD_NAME}" "${NEW_MASTER_IPS[$i]}"
+          done
+
+          echo "Run etcd-signer"
+          SURVIVING_MASTER_NODE_SHORT=${SURVIVING_MASTER_NODE%%.*}
+          curl -O https://raw.githubusercontent.com/hexfusion/openshift-recovery/master/manifests/kube-etcd-cert-signer.yaml.template
+          sed "s;__MASTER_HOSTNAME__;${SURVIVING_MASTER_NODE_SHORT};g" kube-etcd-cert-signer.yaml.template > kube-etcd-cert-signer.yaml
```

Comment:
Lets update this to use file from disk now taht MCO has merged https://github.com/openshift/machine-config-operator/blob/master/templates/master/00-master/_base/files/usr-local-share-openshift-recovery-template-kube-etcd-cert-signer-yaml-template.yaml#L3

Reply:
Then use https://github.com/openshift/machine-config-operator/blob/master/templates/master/00-master/_base/files/usr-local-bin-tokenize-signer-sh.yaml

Reply:
Right, I'd rather take care of that in a separate PR so that we could fix few things in parallel